core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -7121,4 +7284,26 @@ private boolean generateUserStoreChain(String userName, Object credential, boole
         // Authenticate using the initial user store from the user store preference list.
         return initialUserStoreManager.authenticate(userName, credential);
     }
+
+    private long doGetCountUsers(String claimUri, String filterValue, UserStoreManager userManager)
+            throws UserStoreException {
+
+        if (userManager instanceof AbstractUserStoreManager) {
+            try {
+                return ((AbstractUserStoreManager) userManager).countUsersWithClaims(claimUri, filterValue);
+
+            } catch (UserStoreException ex) {
```

Comment:
No need to catch the UserStoreException here. Should return propery error code in countUsersWithClaims method.